lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -372,6 +377,10 @@ def __init__(self, distribution, desired_units,
                     "A default initial guess will be used without "
                     "estimating coefficients from a linear model.")
                 warnings.warn(msg, ImportWarning)
+        else:
+            statsmodels_found = True
+            import statsmodels.api as sm
+            self.sm = sm
         self.statsmodels_found = statsmodels_found
 
     def __str__(self):
```

Comment:
This needs updating too?

Reply:
I've updated this `__str__` to be a `__repr__` and added unit tests. I've also added `__repr__` methods to the other classes in ensemble_calibration.py and added associated unit tests.